src/core/EdFi.DataManagementService.Core.External/Backend/UpdateResult.cs

```diff
@@ -34,14 +34,12 @@ public record UpdateFailureIdentityConflict(string ReferencingDocumentInfo) : Up
     /// <summary>
     /// A transient failure due to a transaction write conflict
     /// </summary>
-    /// <param name="FailureMessage">A message providing failure information</param>
-    public record UpdateFailureWriteConflict(string FailureMessage) : UpdateResult();
+    public record UpdateFailureWriteConflict() : UpdateResult();
```

Comment:
Removed message because it should be entirely core-provided on a write conflict.